src/kani-compiler/src/codegen_cprover_gotoc/codegen/rvalue.rs

```diff
@@ -467,8 +493,8 @@ impl<'tcx> GotocCtx<'tcx> {
                     } else {
                         relative_discr
                             .clone()
-                            .cast_to(Type::unsigned_int(64))
-                            .le(Expr::int_constant(relative_max, Type::unsigned_int(64)))
+                            .cast_to(Type::unsigned_int(128))
```

Comment:
Why do you need this?

Reply:
Good point. It is not needed. I removed it and used `relative_discr`'s type for `relative_max`.